.oca-port.json

```diff
@@ -0,0 +1,11 @@
+{
+  "14.0": {
+    "16.0-mig-l10n_fr_hr_check_ssnid": {
+      "l10n_fr_hr_check_ssnid": {
+        "blacklist_pull_requests": {
+          "302": true
```

Comment:
In fact why did you blacklisted this PR ? 
#302 is fixing a lot of things.

(not sure, though. I don't understand very well this new oca convention).

Reply:
You are probably right in the last version i pushed, there is no oca port. to check.